src/renderer/widgets/CreateWallet/ui/MultisigWallet/MultisigWalletWizard.tsx

```diff
@@ -0,0 +1,65 @@
+import { useToggle } from '@shared/lib/hooks';
+import { DEFAULT_TRANSITION } from '@shared/lib/utils';
+import { MultisigWallet } from './MultisigWallet';
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  onComplete: () => void;
+};
+
+// const enum Step {
```

Comment:
Add comment for commented code or remove it, because now this component looks useless

Reply:
same here, this will be used shortly, when I tackled the flexible multisig implementation. I discussed this with Yaroslav and I think it's better to just leave it there.